packages/common/src/routers/submissionRouter.ts

```diff
@@ -0,0 +1,24 @@
+import { Router, json, urlencoded } from 'express';
+import multer from 'multer';
+
+import { auth } from '../middleware/auth.js';
+
+import { Dependencies } from '../config/config.js';
+import submissionControllers from '../controllers/submissionController.js';
+
+const router = (dependencies: Dependencies): Router => {
+	const upload = multer({ dest: '/tmp' });
+
+	const router = Router();
+	router.use(urlencoded({ extended: false }));
+	router.use(json());
+
+	router.get('/category/:categoryId', auth, submissionControllers(dependencies).listActive);
+
+	router.post('/category/:categoryId/upload', upload.array('files'), submissionControllers(dependencies).upload);
+
+	router.post('/category/:categoryId/commit/:id', auth, submissionControllers(dependencies).commit);
+	return router;
+};
```

Comment:
Creating 3 routers for Submission. Although only `upload` route has implementation. The others return error 501.

Reply:
what will be the difference between uploading and committing?

Reply:
`Upload` puts data in submission for validation and allows to fix errors, add multiple entities, etc. . `Commit` is an user confirmation that the submission is completed.